R/table-format.R

```diff
@@ -153,18 +153,32 @@ pad_prop <- function(x, digits, fmt_small = TRUE, keep_zero = FALSE,
   digits <- check_pos_int(digits)
   output <- check_output(output)
   new_x <- fmt_prop(x, digits = digits, fmt_small = fmt_small,
-                    keep_zero = keep_zero)
+                    keep_zero = keep_zero, output = output)
   new_x[is.na(new_x)] <- "NA"
 
-  if (any(stringr::str_detect(new_x, "^<|^>")) &
-      !all(stringr::str_detect(new_x, "^<|^>"))) {
-    pad <- ifelse(output == "latex", 4, 3)
-    new_x <- dplyr::case_when(stringr::str_detect(new_x, "^<|^>") ~
-                                paste0(new_x, paste(rep("\\ ", pad),
-                                                    collapse = "")),
-                              TRUE ~ new_x)
+  if(is_html_output()) {
```

Comment:
Why use `is_html_output()` here? In the function definition we allow the user to specify and output format. Shouldn't we respect that specification here?